packages/service-library/src/servicelib/volumes_utils.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from typing import Final, Optional
+
+import aiofiles
+from pydantic import BaseModel, Field, root_validator
+
+HIDDEN_FILE_NAME: Final[str] = ".hidden_do_not_remove"
+AGENT_FILE_NAME: Final[str] = ".agent"
+
+
+class VolumeState(BaseModel):
+    requires_saving: bool = Field(
+        ..., description="if True volume must be saved before closing the sidecar"
+    )
+    was_saved: Optional[bool] = Field(
+        None, description="if True volume was saved when the sidecar is closed, if None"
```

Comment:
This field has three possible states: True, False and None. I do not understand the last ... and the description is half way written

Reply:
also I am not sure why there are 2 booleans?